.pre-commit-config.yaml

```diff
@@ -31,35 +31,74 @@ repos:
 
   # Text file hooks
   - repo: https://github.com/igorshubovych/markdownlint-cli
-    rev: v0.33.0
+    rev: v0.34.0
```

Comment:
All the updates to versions of extant `pre-commit` hooks should go in a separate PR.  @mcdonnnj usually creates such a PR just before we release the Lineage Kraken.

Reply:
Aaah, I see what happened. The Golang pre-commit's README detailed to use `master` as rev But when I ran locally pre-commit complained and recommended `pre-commit autoupdate` which updated the others as well. I'll undo those.

Reply:
Please see commit https://github.com/cisagov/skeleton-generic/pull/133/commits/c47038b51d4b537b9bd64b0299c1b99ca0b6e455